main.go

```diff
@@ -294,11 +296,17 @@ func main() {
 		ExpiredDisruptionGCDelay:              cfg.Controller.ExpiredDisruptionGCDelay,
 	}
 
-	if err := r.SetupWithManager(mgr); err != nil {
+	informerClient := kubernetes.NewForConfigOrDie(ctrl.GetConfigOrDie())
+	kubeInformerFactory := kubeinformers.NewSharedInformerFactoryWithOptions(informerClient, time.Second*30, kubeinformers.WithNamespace(cfg.Injector.ServiceAccount.Namespace))
```

Comment:
Does this 30s parameter mean that we'll resync all pods within the chaos-engineering namespace every 30s in case no events happen on them? Or am I wrong about this parameter?

Reply:
i'll investigate

Reply:
turns out it is. Do you think this value should be higher? I'll go with 180s